pkg/fleetautoscalers/controller_test.go

```diff
@@ -695,6 +720,92 @@ func TestControllerUpdateStatusUnableToScale(t *testing.T) {
 	})
 }
 
+func TestControllerSyncFleetAutoscalerWithCustomSyncInterval(t *testing.T) {
+	t.Run("create fas thread", func(t *testing.T) {
```

Comment:
So I just realised this - shouldn't this have a:

```
	utilruntime.FeatureTestMutex.Lock()
	defer utilruntime.FeatureTestMutex.Unlock()

	assert.NoError(t, utilruntime.ParseFeatures(string(utilruntime.FeatureCustomFasSyncInterval)+"=true"))
```

At the top? I'm actually not quite sure how it works otherwise? 🤔

Reply:
It looks like this is for preventing different feature impacts each other. Anyone, I added it.